SaxionSmartCities/netmobiel-vue-client
# Feature geocoder

A basic implementation of the geocoding service. Serves as an example of how to retrieve (1) suggestions and (2) location with lat and lon from the geocoding service. 

To test:
 * Set the proper env variables (see README)
 * Run application and navigate to search locations.
